src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -249,7 +249,7 @@ public enum LegendsAwakened implements LogicCardInfo {
               bg.em().storeObject("Form_Change",bg.turnCount)
               powerUsed()
 
-              def selected = my.deck.search(max:1,"Select a Deoxys",{it.name.contains("Deoxys") && it.cardTypes.isNot(EX)}) //Deoxys cannot Form Change into Deoxys ex.
+              def selected = my.deck.search(max:1,"Select a Deoxys",{it.name.contains("Deoxys") && it.cardTypes.isNot(EX) && it.cardTypes.isNot(LEGEND) && it.cardTypes.isNot(POKEMON_EX) && it.cardTypes.isNot(POKEMON_GX) && it.cardTypes.isNot(POKEMON_V)}) //Deoxys cannot Form Change into Deoxys ex.
```

Comment:
Using `it.name == "Deoxys"` or something similar should resolve needing to continue to add to this every time a new card type is released.

Reply:
The problem is the name isn't deoxys, but deoxys attack forme, etc. That was why the card was bugged in the first place. I can probably simplify it by refusing rulebox pokemon instead of the giant list though. That should prevent said issue.

Reply:
It won't. Rule-boxes aren't what you think they are.

Reply:
`it.name == "Deoxys" || (it.name.contains("Deoxys") && it.name.contains("Forme")`